tests/bulk_create/models.py

```diff
@@ -51,3 +53,34 @@ class TwoFields(models.Model):
 
 class NoFields(models.Model):
     pass
+
+
+class NullableFields(models.Model):
+    """contains all available fields specified in db.backends.oracle.BulkInsertMapper"""
+    big_int_filed = models.BigIntegerField(null=True, default=1)
+    binary_field = models.BinaryField(null=True, default=b'data')
+    boolean_field = models.BooleanField(null=True, default=False)
+    date_field = models.DateField(null=True, default=datetime.date.today())
+    datetime_field = models.DateTimeField(null=True, default=datetime.datetime.now())
+    decimal_field = models.DecimalField(null=True, max_digits=2, decimal_places=1, default='1.1')
+    float_field = models.FloatField(null=True, default=3.2)
+    integer_field = models.IntegerField(null=True, default=2)
+    null_boolean_field = models.NullBooleanField(null=True, default=False)
+    positive_integer_field = models.PositiveIntegerField(null=True, default=3)
+    positive_small_integer_field = models.PositiveSmallIntegerField(null=True, default=4)
+    small_integer_field = models.SmallIntegerField(null=True, default=5)
+    time_field = models.TimeField(null=True, default=datetime.datetime.now().time())
+    # Not listed in BulkInsertMapper because works correctly
```

Comment:
The fields that aren't broken could be added in a separate commit just to make it clear which fields are fixed.